src/screens/LearnWebViewScreen.tsx

```diff
@@ -0,0 +1,122 @@
+import { useRoute } from '@react-navigation/core';
+import React, { useCallback, useEffect, useRef, useState } from 'react';
+import { Share, StatusBar } from 'react-native';
+import { WebView } from 'react-native-webview';
+import { useDimensions } from '@/hooks';
+import { useTheme } from '@/theme';
+import ActivityIndicator from '@/components/ActivityIndicator';
+import Spinner from '@/components/Spinner';
+import { SlackSheet } from '@/components/sheet';
+import { Box, Text } from '@/design-system';
+import { sharedCoolModalTopOffset } from '@/navigation/config';
+import { globalColors } from '@/design-system/color/palettes';
+import { ButtonPressAnimation } from '@/components/animations';
+import { IS_ANDROID } from '@/env';
+import { analytics } from '@/analytics';
+
+const HeaderHeight = 60;
+
+export default function LearnWebViewScreen() {
+  const {
+    params: { cardType, category, title, url },
+  }: // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  any = useRoute();
+  const { isDarkMode } = useTheme();
+  const { height: deviceHeight, isSmallPhone } = useDimensions();
+  const [webViewHeight, setWebViewHeight] = useState(0);
+  const startTime = useRef(Date.now());
+
+  useEffect(
+    () => () => {
+      analytics.track('Learn card opened', {
+        durationSeconds: (Date.now() - startTime.current) / 1000,
+        url,
+        title,
+        category,
+        cardType,
+      });
+      return;
+    },
+    [cardType, category, title, url]
+  );
+
+  const onPressShare = useCallback(async () => {
+    await Share.share({ url });
+    analytics.track('Learn card web view share modal opened', {
```

Comment:
same same typed is mad cool 


prob worth adding a new category for "cards" similar to swaps